packages/asset-service/src/service/AssetService.test.ts

```diff
@@ -133,7 +142,18 @@ describe('AssetService', () => {
   })
 
   describe('description', () => {
+    it('should return the overridden description if it exists', async () => {
+      const assetService = new AssetService(assetFileUrl)
+
+      await expect(assetService.description({ asset: EthAsset })).resolves.toEqual(
+        'overriden description'
+      )
+    })
+
     it('should return a string if found', async () => {
+      const assetDescriptions = descriptions as Record<string, string>
+      delete assetDescriptions[EthAsset.caip19]
```

Comment:
We can also mock it inside the overriden test but:

- We will need to use `require`, so we will need to ignore the typescript error
- Looks less maintainable
- Harder to understand

Reply:
If you have a better solution, don't hesitate, would love to learn it!